src/main/java/reposense/parser/ArgsParser.java

```diff
@@ -25,6 +26,10 @@
     private static final String MESSAGE_SINCE_DATE_LATER_THAN_UNTIL_DATE =
             "\"Since Date\" cannot be later than \"Until Date\"";
 
+    private static String generateDefaultReportName() {
+        return Constants.DEFAULT_REPORT_NAME_FORMAT.format(new Date());
```

Comment:
leave the constant in this class, we should deprecate `Constants` because it doesn't really helps in terms of readability.